doc/man3/DEFINE_STACK_OF.pod

```diff
@@ -213,6 +220,9 @@ index is out of range.
 sk_TYPE_new() and sk_TYPE_new_null() return an empty stack or B<NULL> if
 an error occurs.
 
+sk_TYPE_new_reserve() returns a stack with allocation of the required
+memory or B<NULL> if an error occurs.
```

Comment:
I think this could be included with sk_TYPE_new rather than being spelt out separately.  That there is additional memory allocated is mentioned in the description and isn't relevant to the returns section.

> sk_TYPE_new(), sk_TYPE_new_null() and sk_TYPE_new_reserve() return an empty stack or B<NULL> if an error occurs.

Reply:
Then you might need to indicate the difference of returned stack among those `new` APIs, so how about:

> sk_TYPE_new(), sk_TYPE_new_null() and sk_TYPE_new_reserve() return an empty stack with or without preallocated memory, or B<NULL> if an error occurs.

Reply:
That's okay too.  The preallocation is mentioned above in the function description.  I don't see a hard requirement for it to be mentioned again here.

Reply:
Hmmm, sounds reasonable.